### DISCUSSION TITLE
连接会在整点 和 40分的时候掉线一下

### DISCUSSION THREAD

**Discussion by Klosw** [natfrp/wiki#148] [水深火热] [answered]
我问问 连接会在整点 和 40分的时候掉线一下  你们也是这样吗

**Comment by tidymoon** [comment]
或许应该是你自己的网络波动？

**Reply by Klosw** [reply]
不太像

**Reply by tidymoon** [reply]
若您自己的网络没有问题，则大概为运营商抽风之类的~

**Reply by Klosw** [reply]
是openwrt frp  有个 40分钟定时注册功能导致的 现在已经修改配置后就不掉线 了

**Comment by fakeboboliu** [comment]
看起来很神奇，建议先检查下家里的路由器或者光猫有没有定期重启的功能，或者电脑有没有什么奇妙的定时任务？

**Reply by Klosw** [reply]
没有定期重启 就是会掉线  很奇怪 所以想问问你们 有没有这种情况 哈哈哈

**Reply by Klosw** [reply]
是openwrt frp  有个 40分钟定时注册功能导致的 现在已经修改配置后就不掉线 了

**Comment by Klosw** [comment, marked as answer]
是openwrt frp 服务注册间隔默认为 40分钟导致   现在已经修改配置后就不掉线了

**Reply by Chenime** [reply]
问一下，是怎么解决的

**Reply by Chenime** [reply]
我现在也是这样，

**Reply by Klosw** [reply]
![image](https://user-images.githubusercontent.com/13143959/192681832-9086d6b8-1c94-48ec-8f40-4cb902ed9ae4.png)
我没有用樱花提供的 frpc文件 修改的是 openwrt配置文件这里有个注册时间 填0 就没有问题 ,如果你和我不是一样的我也不知道 看看你有没有填类似的参数吧

**Reply by Klosw** [reply]
你的问题解决了吗

**Reply by Chenime** [reply]
我直接重置了系统,现在没得事了

**Comment by 949047115** [comment]
这个东西在哪打开呀大佬

**Reply by Klosw** [reply]
啥??

**Reply by Klosw** [reply]
直接用 樱花提供的 frpc文件 就好了

**Reply by 949047115** [reply]
frpc文件在哪嘞 找不到呢

**Reply by Klosw** [reply]
> frpc文件在哪嘞 找不到呢
在这里下载
https://www.natfrp.com/tunnel/download

**Reply by Klosw** [reply]
你现在要在什么系统上使用内网穿透 , 你遇到什么问题了 .....

**Reply by 949047115** [reply]
会掉线和你说的问题一样

**Reply by Klosw** [reply]
你怎么使用的

**Reply by 949047115** [reply]
就是用来mc联机 和朋友玩的时候老是不断重连 然后朋友又得重新进房

**Reply by 949047115** [reply]
谢谢你愿意回复我 凑合玩也没事啦

**Reply by Klosw** [reply]
台式是吗 .我也挂了一个mc 一直没有断过呀

**Reply by Klosw** [reply]
你那个是固定时间掉线吗

**Reply by 949047115** [reply]
10:40:50 I Tunnel/eeee [c09e2a01] 开始不断线重连

**Reply by Klosw** [reply]
用的是 frp 客户端是 他官方的客户端 还是自己下载的 frp

**Reply by 949047115** [reply]
https://getfrp.sh/l/SakuraLauncher.exe在这里面下的

**Reply by Klosw** [reply]
我研究一下哈

**Reply by 949047115** [reply]
没事不着急的

**Reply by fakeboboliu** [reply]
> 10:40:50 I Tunnel/eeee [c09e2a01] 开始不断线重连

一般是网络问题，请考虑换个节点再试，同时对游戏等稳定性要求高的场景，不要用无线，不要用热点

**Reply by 949047115** [reply]
不过我删掉那个隧道重建个隧道后他又不掉了

**Reply by Klosw** [reply]
我先挂个 几个小时看看 正常 官方的东西 不怎么会出问题呀

**Reply by 949047115** [reply]
哈哈哈哈哈这也太麻烦你了

**Reply by Klosw** [reply]
主要是 我家里的那个 小的 linux系统 挂了玩游戏反正几乎没断连过 ..

**Reply by Klosw** [reply]
> 哈哈哈哈哈这也太麻烦你了
你家里是啥网络呀

**Reply by 949047115** [reply]
有线网 电信的

**Reply by Klosw** [reply]
我也看到什么不断线重连了 哈哈哈

**Reply by Klosw** [reply]
不过 不频繁 一晚上就出现了一次

**Reply by Klosw** [reply]
好像固定 3点会重连  其他时候都没有掉过 哦
